src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int colorCount = Color.values().length;
+        Random random = new Random();
```

Comment:
lets make Random a class level variable to avoid creating a new instance each time this method is invoked